ast/functions.go

```diff
@@ -164,52 +164,53 @@ const (
 	YearWeek         = "yearweek"
 
 	// string functions
-	ASCII          = "ascii"
-	Bin            = "bin"
-	Concat         = "concat"
-	ConcatWS       = "concat_ws"
-	Convert        = "convert"
-	Elt            = "elt"
-	ExportSet      = "Export"
-	Field          = "field"
-	Format         = "format"
-	FromBase64     = "from_base64"
-	InsertFunc     = "insert_func"
-	Instr          = "instr"
-	Lcase          = "lcase"
-	Left           = "left"
-	Length         = "length"
-	LoadFile       = "load_file"
-	Locate         = "locate"
-	Lower          = "lower"
-	Lpad           = "lpad"
-	LTrim          = "ltrim"
-	MakeSet        = "make_set"
-	Mid            = "mid"
-	Oct            = "oct"
-	Ord            = "ord"
-	Quote          = "quote"
-	Repeat         = "repeat"
-	Replace        = "replace"
-	Reverse        = "reverse"
-	Right          = "right"
-	RTrim          = "rtrim"
-	Space          = "space"
-	Strcmp         = "strcmp"
-	Substring      = "substring"
-	Substr         = "substr"
-	SubstringIndex = "substring_index"
-	ToBase64       = "to_base64"
-	Trim           = "trim"
-	Upper          = "upper"
-	Ucase          = "ucase"
-	Hex            = "hex"
-	Unhex          = "unhex"
-	Rpad           = "rpad"
-	BitLength      = "bit_length"
-	CharFunc       = "char_func"
-	CharLength     = "char_length"
-	FindInSet      = "find_in_set"
+	ASCII           = "ascii"
+	Bin             = "bin"
+	Concat          = "concat"
+	ConcatWS        = "concat_ws"
+	Convert         = "convert"
+	Elt             = "elt"
+	ExportSet       = "export_set"
+	Field           = "field"
+	Format          = "format"
+	FromBase64      = "from_base64"
+	InsertFunc      = "insert_func"
+	Instr           = "instr"
+	Lcase           = "lcase"
+	Left            = "left"
+	Length          = "length"
+	LoadFile        = "load_file"
+	Locate          = "locate"
+	Lower           = "lower"
+	Lpad            = "lpad"
+	LTrim           = "ltrim"
+	MakeSet         = "make_set"
+	Mid             = "mid"
+	Oct             = "oct"
+	Ord             = "ord"
+	Quote           = "quote"
+	Repeat          = "repeat"
+	Replace         = "replace"
+	Reverse         = "reverse"
+	Right           = "right"
+	RTrim           = "rtrim"
+	Space           = "space"
+	Strcmp          = "strcmp"
+	Substring       = "substring"
+	Substr          = "substr"
+	SubstringIndex  = "substring_index"
+	ToBase64        = "to_base64"
+	Trim            = "trim"
+	Upper           = "upper"
+	Ucase           = "ucase"
+	Hex             = "hex"
+	Unhex           = "unhex"
+	Rpad            = "rpad"
+	BitLength       = "bit_length"
+	CharFunc        = "char_func"
+	CharLength      = "char_length"
+	CharacterLength = "character_length"
```

Comment:
We don't need to add it, in parser `character_length` is parsed to `char_length`.